hardhat-tests/test/internal/hardhat-network/stack-traces/test.ts

```diff
@@ -191,22 +191,6 @@ function defineDirTests(dirPath: string, compilerOptions: SolidityCompiler) {
       describe(description, function () {
         defineTest(dirPath, testDefinition, sources, compilerOptions);
       });
-
-      if (process.env.HARDHAT_NETWORK_TESTS_WITH_OPTIMIZATIONS !== undefined) {
```

Comment:
Is there still value in testing with different optimizer `runs` config? If there is, we can setup our workflow to test that separately or simply always define these tests with different numbers.

Reply:
I'm not sure how much value there is, but we weren't doing it so I guess it's not _that_ important.

If needed, we can add entries to `compilers-list.ts` with different runs. I think that should work with the current setup.